clients/src/test/java/org/apache/kafka/clients/CommonClientConfigsTest.java

```diff
@@ -82,4 +102,21 @@ public void testExponentialBackoffDefaults() {
         assertEquals(Long.valueOf(123L),
                 reconnectBackoffSetConf.getLong(CommonClientConfigs.RECONNECT_BACKOFF_MAX_MS_CONFIG));
     }
+
+    @Test
+    public void testInvalidSaslMechanism() {
```

Comment:
This is testing `SaslConfigs.SASL_MECHANISM`, so would it be better to move this test in `SaslConfigsTest`

Reply:
If we move this testCase to `SaslConfigsTest`, the checkstyle of `SaslConfigsTest` will fail.
`Disallowed import - org.apache.kafka.clients.CommonClientConfigs.[ImportControl]`

Reply:
You can modify [this file](https://github.com/apache/kafka/blob/trunk/checkstyle/import-control.xml) to prevent those errors.

Reply:
Actually since this is testing logic that is in `CommonClientConfigs`, maybe it's better to keep this here.

Reply:
Thanks @C0urante  and @mimaison .
I also think it's better to keep this here.